README.md

```diff
@@ -1,321 +1,44 @@
 # Osrs-Wrapper
 Simple wrapper around the [Old School Runescape](http://oldschool.runescape.com/) API
 
-### Dependencies
-* [Request](https://www.npmjs.com/package/request)
-* [Request-Promise](https://www.npmjs.com/package/request-promise)
+## Dependencies
+* [gaxios](https://www.npmjs.com/package/gaxios)
 
-### Installation 
+## Installation 
 ```
 $ npm install --save osrs-wrapper
 ```
 
 ## Usage
-
 This wrapper interacts with the following Old-School APIs:
 * Hiscores
 * Grand Exchange
 
-#### Hiscores
-##### getPlayer(username, type)
-Used to lookup a players skills and minigame stats.  Accepts:
-* Normal (Default)
-* Ironman
-* Hardcore Ironman
-* Ultimate Ironman
-* Deadman
-* Deadman Seasonal
-
-An example of getting a player's hiscore data:
-```javascript
-const osrs = require("osrs-wrapper");
+## Api Reference
 
-osrs.hiscores.getPlayer("Macca P")
-    .then(player => {
-        console.log(JSON.stringify(player, null, 2));
-    });
-```
-Again, but with Hardcore Ironman Hiscores
-```javascript
-const osrs = require("osrs-wrapper");
+### **getHiscores**
+```ts
+import { getHiscores } from './osrs-wrapper/hiscores/hiscores.module';
```

Comment:
Would be a bit clearer like a third party would use it: `import { getHiscores } from 'osrs-wrapper'`

Reply:
Thanks! Updated.